movemap.cpp

```diff
@@ -3,23 +3,23 @@
 #include "movemap.h"
 
 MoveMap::MoveMap(std::vector<std::tuple<Element, Element, Element>> moves) {
-    for (auto element : ELEMENTLIST) {
-        _inputs.insert({element, {}});
-        _outputs.insert({element, {}});
-        for (auto element2 : ELEMENTLIST) {
-            _inputs.at(element).insert({element2, 0});
-            _outputs.at(element).insert({element2, 0});
-        }
+  for (auto element : ELEMENTLIST) {
+    _inputs.insert({element, {}});
+    _outputs.insert({element, {}});
+    for (auto element2 : ELEMENTLIST) {
+      _inputs.at(element).insert({element2, 0});
+      _outputs.at(element).insert({element2, 0});
     }
+  }
 
-    for (auto move : moves) {
-        Element gemType = std::get<0>(move);
-        Element sourceSphere = std::get<1>(move);
-        Element destSphere = std::get<2>(move);
+  for (auto move : moves) {
+    Element gemType = std::get<0>(move);
+    Element sourceSphere = std::get<1>(move);
+    Element destSphere = std::get<2>(move);
 
-        _inputs.at(sourceSphere).at(gemType)++;
-        _outputs.at(destSphere).at(gemType)++;
-    }
+    _inputs.at(sourceSphere).at(gemType)++;
+    _outputs.at(destSphere).at(gemType)++;
+  }
 }
 
 int MoveMap::inputCount(const int sphereType, const int gemType) const {
```

Comment:
The PR says that this is for comment changes. Please keep code formatting separate where possible, especially since there are no comment changes in this file.